client/src/containers/ListEvents.js

```diff
@@ -1,128 +1,145 @@
 // @flow
```

Comment:
@isaadabbasi i already mentioned we need to keep any formatting things in seperate PR, I dont know what to review in this